runtime/src/test_utils.rs

```diff
@@ -360,6 +360,11 @@ impl<BS> MockRuntime<BS> {
             actor_exit: Default::default(),
         }
     }
+
+    /// Enable logging to enviornment. Returns error if already init.
```

Comment:
Why do we care about the error?

Reply:
I'd just return nothing.

Reply:
We don't really care about the actual error, its just more flexible this way (and it is for debugging so flexibility is nice)